### PR TITLE
Create Makefile

### PR DESCRIPTION
**Pull request by brunodrugowick**
Adds a simple Makefile to help in patching and training with overridable defaults.

### CODE REVIEW

**Comment by brunodrugowick** [comment]
This can, and should, be improved in the future. This basic version helps to run basic commands for patch generation (`make patch`) and training (`make train`).

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.PHONY = patch train
+
+CVAT_EXPORT_ROOT = 1
```
Comment (joaoherrera) [inline Makefile]: Suggestion: Change this to the project name on CVAT (i.e. Manual)

**Review by joaoherrera — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (brunodrugowick) [reply Makefile]: This is just a default value. You can override any Makefile variable, for example:

```
make <target> CVAT_EXPORT_ROOT=Manual
```

Do you mean set the default to `Manual` or something?

**Comment by joaoherrera** [comment]
Yeap, set the default to manual

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

File: Makefile (hunk 1 above)
Comment (brunodrugowick) [reply Makefile]:
```suggestion
CVAT_EXPORT_ROOT = "Manual"
```

**Pull request merged**